src/main/java/app/coronawarn/verification/portal/service/TeleTanService.java

```diff
@@ -10,8 +10,11 @@
 @Service
 public class TeleTanService {
 
-  @Autowired
-  private VerificationServerClient verificationServerClient;
+  private final VerificationServerClient verificationServerClient;
```

Comment:
same like ...
Thx for ur constribution. I agree about distaste for field injection, but finally I would recommend the usage of @RequiredArgsConstructor annotation - will cause a constructor being added during the compilation process taking all final fields as parameters and the additional @NonNull annotation will cause the parameter be checked for null as well.